packages/db/package.json

```diff
@@ -62,7 +62,7 @@
     "@libsql/client": "^0.4.3",
     "async-listen": "^3.0.1",
     "deep-diff": "^1.0.2",
-    "drizzle-orm": "^0.28.6",
+    "drizzle-orm": "0.29.5",
```

Comment:
Any specific notes as to why this is pinned to an exact version?

Reply:
Ah, it shouldn't be pinned! Good catch